lib/compilers/golang.ts

```diff
@@ -185,14 +212,14 @@ export class GolangCompiler extends BaseCompiler {
         result.asm = this.convertNewGoL(out);
         result.stderr = null;
         result.stdout = utils.parseOutput(logging, result.inputFilename);
-        return [result, ''];
+        return Promise.all([result, '']);
```

Comment:
This change looked sketchy at first, as the baseCompiler wants this to be a `Promise<[any, any]>`. Afaik this is how you get it even though `result` is already there